src/library/Uno.Material/Styles/Controls/Divider.xaml

```diff
@@ -0,0 +1,45 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:converters="using:Uno.Material.Converters"
+					xmlns:controls="using:Uno.Material.Controls"
+					xmlns:toolkit="using:Uno.UI.Toolkit">
+
+	<ResourceDictionary.MergedDictionaries>
+		<ResourceDictionary Source="../Application/Colors.xaml" />
+	</ResourceDictionary.MergedDictionaries>
+
+	<x:Double x:Key="MaterialDividerHeight">1</x:Double>
+
+	<Style x:Key="MaterialDividerStyle"
+		   TargetType="controls:Divider">
+
+		<Setter Property="Foreground"
+				Value="{StaticResource MaterialOnSurfaceFocusedBrush}" />
+		<Setter Property="SubHeaderForeground"
+				Value="{StaticResource MaterialOnSurfaceLowBrush}" />
+
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="controls:Divider">
+					<Grid>
+						<Grid.RowDefinitions>
+							<RowDefinition Height="auto" />
+							<RowDefinition Height="auto" />
+						</Grid.RowDefinitions>
+
+						<Rectangle Fill="{TemplateBinding Foreground}"
+								   Height="{StaticResource MaterialDividerHeight}" />
+
+						<TextBlock Text="{TemplateBinding SubHeader}"
+								   Visibility="{Binding SubHeader, RelativeSource={RelativeSource Mode=TemplatedParent}, Converter={StaticResource Material_EmptyToCollapsed}, FallbackValue=Collapsed}"
+								   Style="{StaticResource MaterialCaption}"
+								   Foreground="{TemplateBinding SubHeaderForeground}"
+								   Margin="0,4,0,0"
+								   Grid.Row="1" />
```

Comment:
@MatFillion / @XiaotianNetlift Do we want a TextBlock or a ContentPresenter here ? In case someone wants to put something more complex than just a text?

Reply:
As discussed, for now we do not need a contentpresenter. A textblock is enough